tmux-gruvbox-dark.conf

```
@@ -36,23 +30,21 @@ set-option -g display-panes-colour colour237 #bg1
 set-window-option -g clock-mode-colour colour109 #blue
 
 # bell
-set-window-option -g window-status-bell-style fg=colour235,bg=colour167 #bg, red
-
+set-window-option -g window-status-bell-style bg=colour167,fg=colour235 # bg=red, fg=bg
 
 ## Theme settings mixed with colors (unfortunately, but there is no cleaner way)
-set-option -g status-attr "none"
+set-option -g status-style attr=none
 set-option -g status-justify "left"
-set-option -g status-left-attr "none"
+set-option -g status-left-style attr=none
 set-option -g status-left-length "80"
-set-option -g status-right-attr "none"
+set-option -g status-right-status attr="none"
```

`Invalid option: status-right-status`

---

It should be `status-right-style` I think.

```suggestion
set-option -g status-right-style none
```